pkg/osquery/extension.go

```diff
@@ -548,6 +548,26 @@ func (e *Extension) writeLogsLoopRunner() {
 	}
 }
 
+// numberOfBufferedLogs returns the number of logs buffered for a given type.
+func (e *Extension) numberOfBufferedLogs(typ logger.LogType) (int, error) {
```

Comment:
total nit, but since this func is only used by the the tests, you could put it in the `extension_test.go` file and the build will be ever so slightly smaller =)

Reply:
Yeah.... I wasn't sure if we'd want to expose that later. 

But I was thinking about how it would be interesting to put this sort of thing into the `_test` file. Especially with a `must` form, or one that takes `t` and does the error handling internally.